### ISSUE TITLE
Add a /health endpoint

### ISSUE THREAD

**Issue by icc-garciaju** [openwallet-foundation/didcomm-mediator-credo#13]
It could be nice to have a /health enpoint in order to be able to monitor the status of the mediator.

Thanks in avance and thanks for your great work.

**Comment by TimoGlastra** [comment]
Agreed, that would be a good addition. should it be exposed on a separate port? You probably don't want this to be exposed to the public, where the endpoint for DIDcomm should be exposed.

This can of course be managed by routing based on path.

Are you willing to open a PR for these changes?

**Comment by icc-garciaju** [comment]
Having it on another port will suffice in most cases.

If you deploy it on a kubernetes cluster, and want to monitor it externally, you can use a path on the ingress to redirect it to another port. For kubernetes healthcheck, it won't make any difference.
If you deploy it using docker, you can use a reverse proxy to protect it based on source IP.

But having it on the same port, would be easier to implement as no other listener has to be added.

Anyway, thank you in advance.

**Comment by jleach** [comment]
Ya 'all talking about health checks for Kubernets? [This exists](https://github.com/openwallet-foundation/didcomm-mediator-credo/pull/38/files#diff-49cc4f0909411e6de89a20d5a97f81c481546b48973288ba70aeb002b0dd4138R104). I tweaked it a bit in PR #32 to just return 202. The arbitrary string doesn't add any value to k8s.

**Comment by TimoGlastra** [comment]
Yeah kunernetes or docker in general 👍
